src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1446,6 +1468,15 @@ private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(INewDocumentActi
             }
         }
 
+        public Stream GetTempStream()
```

Comment:
can we unify this for the source & destination?

Reply:
No